src/libraries/Microsoft.Extensions.Primitives/ref/Microsoft.Extensions.Primitives.cs

```diff
@@ -116,49 +117,49 @@ public partial struct Enumerator : System.Collections.Generic.IEnumerator<Micros
             public void Reset() { }
         }
     }
-    public readonly partial struct StringValues : System.Collections.Generic.ICollection<string>, System.Collections.Generic.IEnumerable<string>, System.Collections.Generic.IList<string>, System.Collections.Generic.IReadOnlyCollection<string>, System.Collections.Generic.IReadOnlyList<string>, System.Collections.IEnumerable, System.IEquatable<Microsoft.Extensions.Primitives.StringValues>, System.IEquatable<string>, System.IEquatable<string[]>
+    public readonly partial struct StringValues : System.Collections.Generic.ICollection<string>, System.Collections.Generic.IEnumerable<string>, System.Collections.Generic.IList<string>, System.Collections.Generic.IReadOnlyCollection<string>, System.Collections.Generic.IReadOnlyList<string>, System.Collections.IEnumerable, System.IEquatable<Microsoft.Extensions.Primitives.StringValues>, System.IEquatable<string?>, System.IEquatable<string[]>
```

Comment:
Should more of these be `string?`?  Are you not allowed to add nulls for example?

Reply:
Fixed